amundsen_application/api/search/v0.py

```diff
@@ -184,6 +185,69 @@ def _map_user_result(result: Dict) -> Dict:
         return results_dict
 
 
-# TODO - Implement
-def _search_dashboard(*, search_term: str, page_index: int, filters: Dict, search_type: str) -> Dict[str, Any]:
-    return {}
+@search_blueprint.route('/dashboard', methods=['GET'])
+def search_dashboard() -> Response:
+    """
+    Parse the request arguments and call the helper method to execute a dashboard search
+    :return: a Response created with the results from the helper method
+    """
+    try:
+        search_term = get_query_param(request.args, 'query', 'Endpoint takes a "query" parameter')
+        page_index = get_query_param(request.args, 'page_index', 'Endpoint takes a "page_index" parameter')
+        search_type = request.args.get('search_type')
+
+        results_dict = _search_dashboard(search_term=search_term, page_index=int(page_index), search_type=search_type)
+
+        return make_response(jsonify(results_dict), results_dict.get('status_code', HTTPStatus.INTERNAL_SERVER_ERROR))
+    except Exception as e:
+        message = 'Encountered exception: ' + str(e)
+        logging.exception(message)
+        return make_response(jsonify(results_dict), HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+@action_logging
+def _search_dashboard(*, search_term: str, page_index: int, search_type: str) -> Dict[str, Any]:
+    """
+    Call the search service endpoint and return matching results
+    Search service logic defined here:
+    TODO ttannis: Update link when amundsensearchlibrary work is merged
+
+    :return: a json output containing search results array as 'results'
+    """
+    # Default results
+    dashboards = {
+        'page_index': page_index,
+        'results': [],
+        'total_results': 0,
+    }
+
+    results_dict = {
+        'search_term': search_term,
+        'msg': '',
+        'dashboards': dashboards,
+    }
+
+    try:
+        url_base = app.config['SEARCHSERVICE_BASE'] + SEARCH_DASHBOARD_ENDPOINT
+        # TODO ttannis: Request consistency to use f'{url_base}?query_term={search_term}&page_index={page_index}'
```

Comment:
Yes the current endpoint from search is formatted differently, so even though things work I was going to ask if there was some ideal format or pattern that we intend to converge on, so that things aren't different every time we add a new endpoint. 

@feng-tao was it intentional to make the dashboard endpoint have a new pattern? If it was intentional is it because we eventually want to make all the endpoints match the new pattern?

Reply:
will take a look tomorrow. if I haven't, please remind me in the standup as it is a bit late now:(